btor/src/boolector_solver.cpp

```diff
@@ -700,7 +722,7 @@ Term BoolectorSolver::apply_prim_op(PrimOp op, Term t0, Term t1, Term t2) const
 
       std::shared_ptr<BoolectorTerm> bt0 =
           std::static_pointer_cast<BoolectorTerm>(t0);
-      result = boolector_apply(btor, &args[0], 1, bt0->node);
+      result = boolector_apply(btor, &args[0], 2, bt0->node);
```

Comment:
I'm assuming this is correct, but it's not immediately clear to me why this 1 was changed to a 2.

Reply:
Ahh yeah, good question. It is correct -- fixing a copy-and-paste bug from before. This is telling boolector how many arguments are in args and this particular case should be applying a function to 2 arguments.